ci-operator/step-registry/vsphere/provision/additional-tags/vsphere-provision-additional-tags-ref.metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+	"path": "vsphere/provision/additional-tags/vsphere-provision-additional-tags-ref.yaml",
```

Comment:
Suggest to use `usertags` instead of `additional-tags` to keep consistent with other platforms.